tlcc/tlcc.go

```diff
@@ -167,11 +174,3 @@ func (t *TrustedLedgerCC) initNewEnclave(genesis []byte) error {
 
 	return nil
 }
-
-func main() {
-	// start chaincode
-	err := shim.Start(New())
-	if err != nil {
-		logger.Errorf("Error starting tlcc: %s", err)
-	}
-}
```

Comment:
Interesting. Does this still work? WIP?

Reply:
Yep, TLCC will be integrated as a "real" system chaincode compiled into the peer directly. That is, no main needed anymore.